spec/factories/user.rb

```diff
@@ -3,7 +3,7 @@
       first_name { Faker::Name.last_name.gsub(/\W/, '').gsub('u0000', '') }
       last_name { Faker::Name.last_name.gsub(/\W/, '').gsub('u0000', '') }
       phone { '+38(023)122-2222' }
-      address { Faker::Address.street_address }
+      address { 'Taxes Temple' }
```

Comment:
what is wrong with faker?

Reply:
I often get mistakes about address: not valid

Reply:
I see. It's because of your custom address validation.